src/test/java/ca/corefacility/bioinformatics/irida/ria/unit/web/services/UISequencingRunServiceTest.java

```diff
@@ -0,0 +1,215 @@
+		objectService = mock(SequencingObjectService.class);
+		messageSource = mock(MessageSource.class);
+		service = new UISequencingRunService(runService, objectService, messageSource);
+
+		user1 = new User(1L, "mirabel", "mirabel@casita.ca", "Password1!", "Mirabel", "Madrigal", "1234");
+		user2 = new User(2L, "isabela", "isabela@casita.ca", "Password1!", "Isabela", "Madrigal", "5678");
+
+		run1 = new SequencingRun(SequencingRun.LayoutType.SINGLE_END, "miseq");
+		run1.setId(1L);
+		run1.setDescription("Sequencing Run 1");
+		run1.setUser(user1);
+
+		run2 = new SequencingRun(SequencingRun.LayoutType.PAIRED_END, "miseq");
+		run2.setId(2L);
+		run2.setDescription("Sequencing Run 2");
+		run2.setUser(user2);
+
+		file1 = new SequenceFile(Path.of("/tmp/file1.fastq"));
+		file1.setId(1L);
+
+		file2 = new SequenceFile(Path.of("/tmp/file2.fastq"));
+		file2.setId(2L);
+
+		file3 = new SequenceFile(Path.of("/tmp/file3.fastq"));
+		file3.setId(3L);
+
+		object1 = new SingleEndSequenceFile(file1);
+		object1.setId(1L);
+
+		object2 = new SequenceFilePair(file2, file3);
+		object2.setId(2L);
+
+		page = new Page<SequencingRun>() {
+			@Override
+			public int getTotalPages() {
+				return 0;
+			}
+
+			@Override
+			public long getTotalElements() {
+				return 0;
+			}
+
+			@Override
+			public <U> Page<U> map(Function<? super SequencingRun, ? extends U> converter) {
+				return null;
+			}
+
+			@Override
+			public int getNumber() {
+				return 0;
+			}
+
+			@Override
+			public int getSize() {
+				return 0;
+			}
+
+			@Override
+			public int getNumberOfElements() {
+				return 0;
+			}
+
+			@Override
+			public List<SequencingRun> getContent() {
+				return List.of(run1, run2);
+			}
+
+			@Override
+			public boolean hasContent() {
+				return false;
+			}
+
+			@Override
+			public Sort getSort() {
+				return null;
+			}
+
+			@Override
+			public boolean isFirst() {
+				return false;
+			}
+
+			@Override
+			public boolean isLast() {
+				return false;
+			}
+
+			@Override
+			public boolean hasNext() {
+				return false;
+			}
+
+			@Override
+			public boolean hasPrevious() {
+				return false;
+			}
+
+			@Override
+			public Pageable nextPageable() {
+				return null;
+			}
+
+			@Override
+			public Pageable previousPageable() {
+				return null;
+			}
+
+			@Override
+			public Iterator<SequencingRun> iterator() {
+				return null;
+			}
+		};
+
+		when(runService.read(anyLong())).thenReturn(run1);
+		when(runService.list(anyInt(), anyInt(), any(Sort.class))).thenReturn(page);
+		when(objectService.getSequencingObjectsForSequencingRun(any())).thenReturn(
+				Stream.of(object1, object2).collect(Collectors.toCollection(HashSet::new)));
+		when(messageSource.getMessage(anyString(), any(), any())).thenReturn(SUCCESS_MESSAGE);
+	}
+
+	@Test
+	void getSequencingRunTest() {
+		SequencingRun response = service.getSequencingRun(run1.getId());
+		assertEquals(response, run1, "Received the correct sequencing run response");
```

Comment:
can you update the assert* calls in this file to pass the expected value as the first param and the actual value as the second param.

Reply:
Please see 63bdb53f0359fb8f3615a8b435f2eaa9ced71509.